src/java.base/share/classes/java/nio/package-info.java

```diff
@@ -50,13 +50,31 @@
  *   href="channels/package-summary.html#multiplex">multiplexed,
  *   non-blocking <br> I/O</a> facility.  </p></li>
  *
+ *   <li><p> <a
+ *   href="file/FileSystem.html"><i>Filesystems</i></a>,
+ *   which provide an interface to a file system <br> hierarchy and a
+ *   factory for objects to access files and <br> other objects in the
+ *   file system.  </p></li>
+ *
+ *   <li><p> <a
+ *   href="file/Path.html"><i>Paths</i></a>,
+ *   which represent a hierarchical sequence of <br> name elements
+ *   used to locate a file in a file system.  </p></li>
+ *
+ *   <li><p> <a
+ *   href="file/attribute/package-summary.html"><i>Attributes</i></a>,
+ *   which represent <i>metadata</i> associated with <br> objects in a
+ *   file system.  </p></li>
```

Comment:
What would you think about reducing this down to "Path, which together with the Files class provide access to files.". The reason is that "FileSystem" isn't something that most developers will use directly and I don't think the attributes package is listed later and I don't think needs to be in the introducing to the abstractions.

Reply:
I agree that the list was too long. Fixed in e72437776a0625f526934294d2df4cab1d848056.